src/game/WorldHandlers/MiscHandler.cpp

```diff
@@ -1273,46 +1273,31 @@ void WorldSession::HandleInspectHonorStatsOpcode(WorldPacket& recv_data)
     SendPacket(&data);
 }
 
-void WorldSession::HandleWorldTeleportOpcode(WorldPacket& recv_data)
+void WorldSession::WorldTeleportHandler(WorldPacket& recv_data)
 {
-    DEBUG_LOG("WORLD: Received opcode CMSG_WORLD_TELEPORT from %s", GetPlayer()->GetGuidStr().c_str());
+    DEBUG_LOG("WORLD: Received opcode CMSG_WORLD_TELEPORT from %s:", GetPlayer()->GetGuidStr().c_str());
 
-    // write in client console: worldport 469 452 6454 2536 180 or /console worldport 469 452 6454 2536 180
```

Comment:
any idea what the 5 parameters are ?
- It might be worth adding an explaination

Reply:
Agreed. I will be committing requested changes soon.

Reply:
Thank you